encrypted-media-respec.html

```diff
@@ -608,11 +608,11 @@
               </p>
               <p>
               Supported configurations, including supported audio and video codecs, may depend on availability of optional capabilities such as
-              <a def-id="distinctive-identifier"></a>s and persistent state. The following algorithm iteratively tries to find a configuration
-              that is both supported and has user consent, each iteration considering a stronger set of restrictions derived from user consent prompts.
+              <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and persistent state. The following algorithm iteratively tries to find a configuration
+              that is both supported and has user consent, each iteration considering a smaller set of possible configurations.
               </p>
               <p>
-              User Agents should persist consent responses at least for the duration of the <a def-id="requestMediaKeySystemAccess"></a>
+              User Agents should reuse earlier consent responses at least for the duration of the <a def-id="requestMediaKeySystemAccess"></a>
```

Comment:
I think we should "when/where appropriate" after "responses", acknowledging that there are some nuances (discussed later).